.github/workflows/fastcdr-test.yml

```diff
@@ -54,37 +56,37 @@ jobs:
           - 'ubuntu-20.04'
           - 'ubuntu-22.04'
           - 'windows-2019'
+          - 'macos-13'
```

Comment:
Fast DDS MacOS official Tier 1 platform is MacOS 10.14 (Mojave). ROS 2 demoted MacOS to Tier 3 with Galactic and since then, they do not specify the platform supported (it is community supported only). I suppose we should update the platform support in Fast DDS.

Reply:
I think we need to discuss this internally and update here on a follow up if needed